lib/views/status.erb

```diff
@@ -71,7 +71,7 @@
         <a class="nav-link active" id="git-tab" href="#git">Git</a>
       </li>
       <li class="nav-item">
-        <a class="nav-link" id="heroku-tab" href="#heroku">Heroku</a>
+        <a class="nav-link <%= "disabled" if !heroku_authenticated? %>" id="heroku-tab" href="#heroku">Heroku</a>
```

Comment:
Maybe we just hide this tab altogether if the application is configured to not want Heroku stuff?

Reply:
@raghubetina good point. Is something like this what you had in mind?

![image](https://user-images.githubusercontent.com/17581658/217315680-bf4e3a49-1f32-405f-9563-f5bfdb602a0a.png)

Reply:
Yes. Eventually perhaps we can add tabs for "Fly.io", "Render", all controlled by the config file. But for now, just blank is least confusing I think.